Admin/Admin.php

```diff
@@ -2926,7 +2926,7 @@ public function getAccessMapping()
     /**
      * {@inheritdoc}
      */
-    public function checkAccess($action, $object = null)
+    public function getAccess()
```

Comment:
@core23 if this lead to less ugly and long twig code, I think it is good to introduce this method. In a later rewrite, we might want to have a dedicated class for that instead of handling this in the `Admin` god object.